Completion/X/Command/_kdeconnect-cli

```diff
@@ -0,0 +1,35 @@
+#compdef kdeconnect-cli
+
+_kdeconnect-cli() {
+  _kdeconnect-cli-devices-id() {
```

Comment:
For Zsh autoloaded functions, you don't need the _kdeconnect-cli() { } wrapping the entire contents of the file. That's superfluous and should be removed.